tests/test_cli.py

```diff
@@ -87,7 +87,7 @@ def _out(self, path: Optional[str] = TEST_OUT) -> str:
     def test_main_help(self):
         result = self.runner.invoke(main, ["--help"])
         out = result.stdout
-        print("STDERR", result.stderr)
+        logging.info("STDERR", result.stderr)
```

Comment:
Maybe switch to a named logger as suggested here ([tutorial](https://docs.python.org/3/howto/logging.html#advanced-logging-tutorial) or [logging cookbook](https://docs.python.org/3/howto/logging-cookbook.html#logging-cookbook)).

To change insert after the import section
```
logger = logging.getLogger(__name__)
```
and then replace all logging.debug, ligging.info etc. by logger.debug, logger.info etc.

linkml-runtime also does it this way ([test_schemaview.py](https://github.com/linkml/linkml-runtime/blob/main/tests/test_utils/test_schemaview.py)).

Reply:
yep!  definitely.  not done with this implementation at all, just first pass at getting the output a bit more grok'able so I can see the breaking errors.

Reply:
I think this refactoring should go in another PR.  The changes here are consistent with the rest of the library and I captured the next step we need to take here: https://github.com/INCATools/ontology-access-kit/issues/838